### PR TITLE
new work started.

### PR DESCRIPTION
**Pull request by codesofmine**
Hello.

### CODE REVIEW

**Review by esalkan — CHANGES_REQUESTED** [review]
Hello @codesofmine welcome our fun repo. Please "Tell us about yourself"

**Review by codesofmine — COMMENTED** [review]
@esalkan okey. now i am going to write some informations about myself.

File: codesofmine.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+// Hello World
```
Comment (esalkan) [inline codesofmine.txt:1]: Write here something about yourself

**Review by esalkan — APPROVED** [review]
All is well now. Welcome again.

**Pull request merged**